ingestion-beam/src/test/java/com/mozilla/telemetry/decoder/DeduplicateTest.java

```diff
@@ -100,6 +101,16 @@ public void testOutput() {
 
     // run RemoveDuplicates
     pipeline.run();
+
+    PCollection<String> nonDedupledIds = pipeline
+        .apply("ids", Create.of(Arrays.asList(newId, duplicatedId)))
+        .apply("create messages", mapStringsToId)
+        .apply("no-op deduplicate", Deduplicate.removeDuplicates(StaticValueProvider.of(null)))
+        .ignoreDuplicates().output().apply("get non-deduped IDs", mapMessagesToId);
+    PAssert.that(nonDedupledIds).containsInAnyOrder(newId, duplicatedId);
```

Comment:
It doesn't appear that we're testing what should be the case here, that is, that the duplicates aren't removed -- there don't appear to be any duplicates in this list.

Reply:
Thank you. You're right. Will fix.

Reply:
Turns out the test _is_ valid, but difficult to reason through. I've added new comments to explain the intent.

By the time this test is running, we've already sent two IDs through Redis to be marked as seen. If we passed the test redis uri here, then IDs that have already been seen would be removed, thus `seenId` would not show up in the output. But the `seenId` passes through when uri is null.